self-tests.yml

```diff
@@ -124,14 +123,11 @@ variables:
             GITHUB_TOKEN: ${github_token}
           command: bash scripts/setup-smoke-config.sh
       - func: run-make
-        vars:
-          target: set-smoke-vars
+        vars: { target: "set-smoke-vars" }
       - func: run-make
-        vars:
-          target: set-smoke-git-config
+        vars: { target: "set-smoke-git-config" }
       - func: run-make
-        vars:
-          target: "${task_name}"
+        vars: { target: "${task_name}" }
```

Comment:
No, should I change back? I just look through them and changed single lines to be the same to try to condense it- we do the others like this for reference

Reply:
I don't think we have a preference between condensed vs. non-condensed since we don't have a YAML style guide. I usually don't use the condensed syntax because as soon as you need to pass more than one key, it starts getting messy, but I don't really care either way. Leaving it or reverting it back doesn't really matter, I was just curious.